docker/scripts/common.sh

```diff
@@ -83,6 +84,25 @@ e2e_test_relayer() {
                         do sleep 1s; done"
 }
 
+e2e_test_op_proof(){
+    local root=$(get_root_dir)
+    local l1_rpc=$1
+    local l2_rpc=$2
+    local network_config=$3
+    local privkey=$4
+    local txhash=$5
+    local relayer=${root}/relayer/relayer-node18-linux-x64
+
+    echo Starting OP relayer message prover...
+    timeout 5m bash -c "until ${relayer} prove-op-message \
```

Comment:
Is the retrying necessary here? Shouldn't this command run succesfully in the first place?

Reply:
This command also waits until the message is ready to be proven, so this can be removed.